gammapy/irf/effective_area_table.py

```diff
@@ -360,8 +364,24 @@ class EffectiveAreaTable2D(object):
     >>> from gammapy.datasets import load_aeff2D_fits_table
     >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
     >>> offset = Angle(0.6, 'degree')
-    >>> energies = Quantity(np.logspace(0, 1, 60), 'TeV')
-    >>> eff_area = aeff2D.evaluate(offset, energies)
+    >>> energy = Quantity(np.logspace(0, 1, 60), 'TeV')
+    >>> eff_area = aeff2D.evaluate(offset, energy)
+
+    Create ARF fits file for a given offest and energy binning:
+
+    >>> import numpy as np
+    >>> from astropy.coordinates import Angle
+    >>> from astropy.units import Quantity
+    >>> from gammapy.irf import EffectiveAreaTable2D
+    >>> from gammapy.datasets import load_aeff2D_fits_table
+    >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
+    >>> offset = Angle(0.43, 'degree')
+    >>> nbins = 50
+    >>> energy = Quantity(np.logspace(0, 1, nbins+1), 'TeV')
```

Comment:
In some places in Gammapy we currently use an [energy_band](https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=energy_band) tuple, in some separate `energy_min` and [energy_max](https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=energy_max&type=Code) variables / parameters.

I'm 50:50 on this, but we probably should choose one and be more consistent.

@mapazarr, @adonath - What do you think?